### PR TITLE
Link to purchase order analysis from purchase registry

### LINKED ISSUE #5333: Link the Purchase Order Analysis report on the Purchase Order Actions menu
We provide an easy way to analyze a purchase order through the Purchase Order Analysis report.  Let's add a link directly to the Purchase Order registry.  Here is where it would go:

image

### PR DESCRIPTION
**Pull request by jniles**
Adds a link from the purchase order registry to the purchase order analysis report.

Closes #5333.

### CODE REVIEW

File: client/src/modules/purchases/list/list.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ angular.module('bhima.controllers')
 PurchaseListController.$inject = [
   '$state', 'PurchaseOrderService', 'NotifyService', 'uiGridConstants',
   'GridColumnService', 'GridStateService', 'SessionService', 'ModalService',
-  'ReceiptModal', 'bhConstants', 'BarcodeService',
+  'ReceiptModal', 'bhConstants', 'BarcodeService', '$httpParamSerializer',
```
Comment (jmcameron) [inline client/src/modules/purchases/list/list.js]: It looks like $httpParamSerializer is not actually used below.

**Review by jmcameron — APPROVED** [review]
The code looks good.
The new link works fine.  
I tested this with the IMCK database and double-checked several types of PO analyses.  Most seemed very good.  However I did notice one oddity:  If you analyze a cancelled PO, the top-level status is "Cancelled" (good), but the status of each item ordered is still listed as "Pending".  See below:
image

**Comment by jmcameron** [comment]
Question: Why did the "chore" items appeared in this PR?

**Comment by jniles** [comment]
> Question: Why did the "chore" items appeared in this PR?

Because I'm bad at rebasing :/

**Comment by jniles** [comment]
Thanks!  I'll get on to fixing these issues.

**Comment by jniles** [comment]
Okay, got the rebase fixed.  I also fixed those issues you pointed out.  Here is the new presentation:

image

**Review by jmcameron — APPROVED** [review]
Looks good.  I'll go ahead and release it.

**Pull request merged**